weblate_web/static/img/delete.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="14px" height="18px" viewBox="0 0 14 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 54.1 (76490) - https://sketchapp.com -->
+    <title>7434ADBA-63E9-4B19-A34C-99D5C7B0B403</title>
+    <desc>Created with sketchtool.</desc>
+    <g id="Web" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Subscriptions" transform="translate(-989.000000, -594.000000)" fill="#2ECCAA" fill-rule="nonzero">
+            <g id="delete" transform="translate(989.000000, 594.000000)">
+                <path d="M14,1 L10.5,1 L9.5,0 L4.5,0 L3.5,1 L0,1 L0,3 L14,3 L14,1 Z M1,16 C1,17.1045695 1.8954305,18 3,18 L11,18 C12.1045695,18 13,17.1045695 13,16 L13,4 L1,4 L1,16 Z" id="Shape"></path>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Is it necessary to have the delete icon as both PNG and SVG?

Reply:
No it isn't, i have deleted the SVG file.

Reply:
It seems to be still present...

Reply:
I have deleted it now in the recent commit.